src/remarkable-ui/components/editors/inputs/NumberField/NumberField.stories.tsx

```
@@ -0,0 +1,94 @@
+};
+
+export const Clearable: Story = {
+  args: {
+    label: undefined,
+    required: false,
+    value: 42,
+    clearable: true,
+  },
+};
+
+export const Error: Story = {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)